### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by 704009905**

### CODE REVIEW

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 1]
```diff
@@ -1,234 +1,234 @@
 #<ELN_LANGFILE_V1_0>
 advanced_machine_block.name=高级机械方块
 cinnabar_ore.name=朱砂矿石
-cheap_chip.name=廉价芯片
-dielectric.name=电介质
-data_logger_print.name=数据记录仪
+cheap_chip.name=低级芯片
+dielectric.name=?绝缘体
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:5]: What happened on that question mark? Did you use non-UTF8 character?

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 2]
```diff
@@ -1,234 +1,234 @@
...
+data_logger_print.name=数据记录器打印
 high_voltage_cable.name=高压电缆
-small_50v_carbon_incandescent_light_bulb.name=50V白炽灯炮
-alloy_plate.name=钨钢合金板
+small_50v_carbon_incandescent_light_bulb.name=小型50V碳白炽灯灯泡
+alloy_plate.name=合金板
 copper_plate.name=铜板
-200v_plate_machine.name=卷板机(200V)
+200v_plate_machine.name=200V制板机
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:11]: Actually "卷板机"（Rolling machine, plate-bending machine）is better.

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 3]
```diff
@@ -1,234 +1,234 @@
...
-animal_filter.name=动物过滤器
-200v_farming_lamp.name=200V农灯灯泡
-200v_economic_light_bulb.name=200V节能灯泡
-electrical_motor.name=电动马达
+on/off_regulator_10%.name=On/OFF调节器 10%
+electrical_probe.name=电子测量器
+animal_filter.name=生物过滤器
+200v_farming_lamp.name=200V农业灯
+200v_economic_light_bulb.name=200V经济灯泡
+electrical_motor.name=电子马达
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:58]: 电子（Electronic），电力（Electrical）
考虑一下。下同。

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 4]
```diff
@@ -1,234 +1,234 @@
...
-street_light.name=路灯
+street_light.name=街灯
 standard_alarm.name=标准警报器
-weak_50v_battery_charger.name=50V弱充电器
+weak_50v_battery_charger.name=低功率50V充电器
 player_filter.name=玩家过滤器
-small_rotating_solar_panel.name=小型可旋转太阳能电板
-lamp_socket_b_projector.name=灯座B版
+small_rotating_solar_panel.name=小型旋转太阳能电池板
+lamp_socket_b_projector.name=灯座B 投影器
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:77]: 多余空格？

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 5]
```diff
@@ -1,234 +1,234 @@
...
 200v_magnetizer.name=200V磁化机
-portable_battery_pack.name=便携电池包
-small_flashlight.name=小型手电筒
+portable_battery_pack.name=便携电池抱
+small_flashlight.name=便携手电筒
 black_brush.name=黑色刷子
-50v_iron_heating_corp.name=50V Iron Heating Corp
-average_electrical_drill.name=普通电钻
+50v_iron_heating_corp.name=50V铁加热组
+average_electrical_drill.name=普通电子钻头
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:83]: line 58
电子／电力。

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 6]
```diff
@@ -1,234 +1,234 @@
...
+small_solar_panel.name=小型太阳能板
 mercury.name=汞
 yellow_brush.name=黄色刷子
 wrench.name=扳手
-signal_trimmer.name=信号调整器
-wireless_switch.name=无线信号开关
-50v_condensator.name=50V Condensator charged at 0%
+signal_trimmer.name=信号微调器
+wireless_switch.name=无线开关
+50v_condensator.name=50V 电容 储量:0%
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:143]: 多余空格？

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 7]
```diff
@@ -1,234 +1,234 @@
...
+removed_from_mod_copper_thermal_cable.name=Removed from mod铜热线缆
 200v_active_thermal_dissipator.name=200V主动散热器
-hub.name=集线器
-single-use_battery.name=一次性电池（满电）
-small_50v_copper_heating_corp.name=Small 50V Copper Heating Corp
-electrical_breaker.name=断路器
-redstone-to-voltage_converter.name=红石-信号转换器
-achievement.openGuide=Wiki之力
-achievement.openGuide.desc=不会Wiki一下么？
+hub.name=枢纽
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:154]: 集线器 is correct. Consider about how to say USB-Hub in Chinese.

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 8]
```diff
@@ -1,234 +1,234 @@
...
+200v_iron_heating_corp.name=200V铁加热组
+200v_turbine.name=200V汽轮机
+industrial_data_logger.name=工业数据记录器
 silicon_plate.name=硅板
 advanced_magnet.name=高级磁铁
-long_suspended_lamp_socket.name=长链灯座
+long_suspended_lamp_socket.name=长悬挂式电灯插座
 signal_switch.name=信号开关
-signal_20h_inductor.name=Signal 20H inductor
+signal_20h_inductor.name=信号20H感应器
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:172]: 20H信号感应器？信号20H感应器？

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 9]
```diff
@@ -1,234 +1,234 @@
...
-capacity_oriented_battery.name=容量升级电池（50%电量）
-electrical_probe_chip.name=电力探测芯片
-thermometer.name=温度表
-on/off_regulator_1%.name=On/OFF Regulator 1%
+cheap_electrical_drill.name=低级电子钻头
+small_50v_economic_light_bulb.name=小型50V经济灯泡
+capacity_oriented_battery.name=容量强化电池 储能:50%
+electrical_probe_chip.name=电子测量芯片
+thermometer.name=温度计
+on/off_regulator_1%.name=On/OFF调节器 1%
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:213]: In such case "On" should be "ON" (i.e. all capitalised). The same as previous entries.

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 10]
```diff
@@ -1,234 +1,234 @@
...
 wireless_signal_receiver.name=无线信号接收器
-auto_miner.name=自动采矿机
-tungsten_ore.name=钨矿 
+auto_miner.name=自动挖矿机
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:215]: In my opinion 采矿机 is better

File: src/main/resources/assets/eln/lang/zh_CN.lang [hunk 11]
```diff
@@ -1,234 +1,234 @@
...
+iron_cable.name=铁线缆
 advanced_chip.name=高级芯片
 water_turbine.name=水轮机
 red_brush.name=红色刷子
-fast_electrical_drill.name=高速电钻
-power_capacitor.name=电力电容器
-copper_ingot.name=铜锭
+fast_electrical_drill.name=快速电子钻头
+power_capacitor.name=电容器
+copper_ingot.name=铜 锭
```
Comment (3TUSK) [inline src/main/resources/assets/eln/lang/zh_CN.lang:231]: 多余空格。

**Comment by 3TUSK** [comment]
Make a rough review for the translation. The rest of them are fine. Feel free to comment me.

**Comment by Baughn** [comment]
I can't review chinese, but—you have merge conflicts now. Can you rebase your change on top of the newest commit, and sort that out?

**Comment by metc** [comment]
Please rebase.

**Comment by metc** [comment]
@704009905 we made some progress with the translations. If you are interested, please create an account on Transiflex and help us to translate the mod in your language.
See https://www.transifex.com/electrical-age/eln/

**Comment by 3TUSK** [comment]
@metc I don't think you guys can find @704009905 again, but I can provide help on that if I have time.

**Comment by metc** [comment]
@3TUSK great, you are welcome ;)

**Pull request closed without merging**